downstream/modules/eda/proc-eda-delete-rulebook-activations.adoc

```diff
@@ -1,8 +1,8 @@
-[id="proc-eda-delete-rulebook-activations"]
+[id="eda-delete-rulebook-activations"]
 
 == Delete rulebook activations
 
-. Select the  btn:[More Actions] icon *{MoreActionsIcon}* next to *Rulebook Activation enabled/disabled* switch.
+. Select the btn:[More Actions] icon *{MoreActionsIcon}* next to *Rulebook Activation enabled/disabled* switch.
```

Comment:
```suggestion
. Select the btn:[More Actions] icon *{MoreActionsIcon}* next to the *Rulebook Activation enabled/disabled* switch.
```

Reply:
Would "toggle" work instead of "switch"?  Both of them make sense.

Reply:
"Switch" was the language I got from Tina in UX but I can change it to "toggle".